osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -107,6 +107,9 @@ public virtual string SettingDescription
         [JsonIgnore]
         public virtual bool RequiresConfiguration => false;
 
+        [JsonIgnore]
+        public virtual bool HidesResumeOverlay => false;
```

Comment:
This cannot exist here. `Mod` is the wrong level of abstraction for this. All mods implement their effects via `IApplicableMod` interfaces, rather than putting stuff into `Mod` directly. All overridable properties on `Mod` are more like metadata about the mod rather and they do not generally control its operation.

Reply:
I removed the prop and set `drawableRuleset.ResumeOverlay` to `null` in `ApplyToDrawableRuleset()`.